__tests__/app.test.js

```diff
@@ -62,36 +62,66 @@ describe("/api/articles/:articles_id", () => {
       .then((response) => {
         const { article } = response.body;
         expect(article).toMatchObject({
-            title: "Living in the shadow of a great man",
-            topic: "mitch",
-            author: "butter_bridge",
-            body: "I find this existence challenging",
-            created_at: "2020-07-09T20:11:00.000Z",
-            votes: 100,
-            article_img_url:
-              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-          });
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
         });
-        
       });
   });
+});
+
+test("404 - responds with Not Found for non-existent id", () => {
+  return request(app)
+    .get("/api/articles/12334404")
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Not Found");
+    });
+});
+
+test("400 - responds with error when id is not a number", () => {
+  return request(app)
+    .get("/api/articles/bosh")
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Invalid Input");
+    });
+});
 
-  test('404 - responds with Not Found for non-existent id', () => {
+describe("GET api/articles", () => {
+  test("200 - repsonds with an article with the correct properties", () => {
     return request(app)
-      .get("/api/articles/12334404")
-      .expect(404)
+      .get("/api/articles")
+      .expect(200)
       .then(({ body }) => {
-        expect(body.msg).toBe("Not Found");
-        
+        body.forEach((article) => {
```

Comment:
When using a forEach we need to assert the length of the array is correct too in (the same test). This is because if an array is not returned (say we change/break our code) and we do not check for the length then the forEach won't run and we will get a false positive pass from the test - when clearly we shouldn't.